lib/vimaly/client.rb

```diff
@@ -54,13 +54,37 @@ def update_ticket(id, other_fields)
       end
     end
 
-    def search_tickets(query_string, state: 'active')
+    def search_tickets(query_string, state: 'active', max_results: 500)
```

Comment:
Hmm, thinking of this parameter `max_results` here. One might think that it's limiting the overall number of tickets the caller would like to get, but in fact it's the per-page setting. Maybe not make this optional at all and only use the max of `500` internally?

Reply:
I did it so I could write a test that would have to loop through multiple times. I can definitely remove this though.